### PR TITLE
Feature/go air

### PR DESCRIPTION
**Pull request by kennybc**
# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] New and existing unit tests pass locally with my changes
- [x] All Lint and CI checks are passing

### CODE REVIEW

**Comment by alexangione419** [comment]
lgtm king :+1:

File: docker-compose.dev.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      dockerfile: Dockerfile.dev
+    ports:
+      - "8080:8080"
+    depends_on:
+      db:
+        condition: service_healthy
+    env_file:
+      - .env
+    environment:
+      - APP_ENVIRONMENT=production
```
Comment (sebytremblay) [inline docker-compose.dev.yaml:15]: Should this be "dev"?

File: backend/.air.toml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  runner = "green"
+  watcher = "cyan"
+
+[log]
+  main_only = false
+  silent = false
+  time = false
+
+[misc]
+  clean_on_exit = false
```
Comment (sebytremblay) [inline backend/.air.toml:46]: explore

File: backend/.air.toml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+[proxy]
+  app_port = 0
```
Comment (sebytremblay) [inline backend/.air.toml:49]: 8080?

File: backend/.air.toml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  enabled = false
+  proxy_port = 0
+
+[screen]
+  clear_on_rebuild = false
```
Comment (sebytremblay) [inline backend/.air.toml:54]: explore

File: backend/.air.toml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  poll = true
+  poll_interval = 500
+  post_cmd = []
+  pre_cmd = []
+  rerun = false
+  rerun_delay = 500
+  send_interrupt = false
+  stop_on_error = false
+
+[color]
```
Comment (sebytremblay) [inline backend/.air.toml:33]: necessary?

File: README.md [hunk 6]
```diff
@@ -66,3 +66,14 @@ VITE_GITHUB_CLIENT_ID=<The client ID of your GitHub OAuth app>
...
+
+## Running locally in dev mode without using Make (due to multi-line env variable issues):
+
+1. Launch Docker Desktop
+2. (On first run) In the repo root: run `docker-compose -f docker-compose.dev.yaml build`
+3. In the repo root: run `docker-compose -f docker-compose.dev.yaml up`
+4. In a new terminal: run `ngrok http --domain={<ngrok public url>} 8080`
+5. In a new terminal: run `cd frontend`
+6. (On first run) In the frontend directory: run `npm i`
+7. In the frontend directory: run `npm run dev`
```
Comment (sebytremblay) [inline README.md:79]: Think the last 3 can be batched into a makefile command

**Review by sebytremblay — CHANGES_REQUESTED** [review]

File: docker-compose.dev.yaml (hunk 1 above)
Comment (kennybc) [reply docker-compose.dev.yaml]: fixed

File: README.md (hunk 6 above)
Comment (kennybc) [reply README.md:79]: this section is specifically not using make

**Review by CamPlume1 — APPROVED** [review]
Nothing that would break local development,

Trusting that it works in your use case

**Review by sebytremblay — APPROVED** [review]

**Pull request merged**